.github/dependabot.yml

```diff
@@ -14,6 +14,12 @@ updates:
     open-pull-requests-limit: 5
     commit-message:
       prefix: "deps"
+    ignore:
+      - dependency-name: "com.squareup.okhttp3:*"
```

Comment:
why we don't want to update OKHTTP?

Reply:
We get different version from the sdk which caused issues in the past. So it is better to update it manually when the sdk is updated.